rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/DiscoverHibernateConfigurationRuleProvider.java

```diff
@@ -130,9 +131,9 @@ private void createHibernateConfigurationModel(GraphRewrite event, EvaluationCon
             sessionFactoryModel.setSessionFactoryProperties(sessionFactoryProperties);
 
             // create the datasource references.
-            if (sessionFactoryProperties.containsKey("hibernate.connection.datasource"))
+            if (sessionFactoryProperties.containsKey("connection.datasource"))
```

Comment:
I am not sure that I understand this. It seems that the documentation uses "hibernate.connection.datasource": https://docs.jboss.org/hibernate/orm/3.6/reference/en-US/html/session-configuration.html

Reply:
I think there is difference between properties in *hibernate.properties* and its usage in code and what is in *hibernate.cfg.xml* where the hibernate prefix is not used

Reply:
Oh, I see that now. :) Thanks!